### PR TITLE
Fix edge case for logging on student submission

### LINKED ISSUE #118: Store correct answer in event log
Currently, the correct answer is not stored anywhere in the event log. This makes analysing the results very difficult.

### PR DESCRIPTION
**Pull request by jleong-openedx**
This handles the case where there is no correct answer.

Closes #118.

### CODE REVIEW

File: ubcpi/ubcpi.py [hunk 1]
```diff
@@ -510,9 +510,18 @@ def record_response(self, answer, rationale, status):
         answers = self.get_answers_for_student()
         stats = self.get_current_stats()
         truncated_rationle, was_truncated = truncate_rationale(rationale)
+        corr_ans_text = ''
+        if self.correct_answer == 3:    # handle scenario with no correct answer
```
Comment (xcompass) [inline ubcpi/ubcpi.py]: Could you remind me where is this "3" coming from?

Comment (jleong-openedx) [inline ubcpi/ubcpi.py]: This indicates that the option at the third index was selected by the instructor, for the selection of the correct answer. Index 3 in the options corresponds to "n/a" or no correct answer.

Comment (xcompass) [inline ubcpi/ubcpi.py]: How do you know it is "3"? I thought the index will change when you add more answers.

Comment (jleong-openedx) [inline ubcpi/ubcpi.py]: Sorry, thanks for the catch, I was thinking only of the default scenario--I have updated the pull request ("n/a" will always be the last-indexed option).

**Comment by xcompass** [comment]
Thanks!

**Pull request merged**